### PR TITLE
Adding kube-burner to cncf landscape

### PR DESCRIPTION
**Pull request by rsevilla87**
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [x] Are you only including a `repo_url` if your project is 100% open source? If so, you need to pick the single best GitHub repository for your project, not a GitHub organization.
* [x] Is your project closed source or, if it is open source, does your project have at least 300 GitHub stars?
* [x] Have you picked the single best (existing) category for your project?
* [x] Does it follow the other guidelines from the [new entries](https://github.com/cncf/landscape#new-entries) section?
* [x] Have you added your SVG to `hosted_logos` and referenced it there?
* [x] Does your logo clearly state the name of the project/product and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [x] Does your project/product name match the text on the logo?
* [x] Have you verified that the Crunchbase data for your organization is correct (including headquarters and LinkedIn)?
* [x] ~30 minutes after opening the pull request, the CNCF-Bot will post the URL for your staging server. Have you confirmed that it looks good to you and then added a comment to the PR saying "LGTM"?

Ref: https://github.com/cncf/toc/issues/1230

### CODE REVIEW

**Comment by rsevilla87** [comment]
LGTM

File: landscape.yml [hunk 1]
```diff
@@ -7113,6 +7113,18 @@ landscape:
...
             crunchbase: https://www.crunchbase.com/organization/xebialabs
+          - item:
+            name: kube-burner
+            homepage_url: https://kube-burner.github.io/kube-burner/
+            project: sandbox
+            repo_url: https://github.com/kube-burner/kube-burner
+            logo: kube-burner.svg
+            crunchbase: https://www.crunchbase.com/organization/red-hat
+            extra:
+              accepted: '2023-18-12'
```
Comment (castrojo) [inline landscape.yml:7124]:
```suggestion
              accepted: '2023-12-18'
```

Comment (castrojo) [reply landscape.yml:7124]: I think this is what you mean. 😄

**Review by castrojo — CHANGES_REQUESTED** [review]

**Comment by rsevilla87** [comment]
Moving this one to draft until we get an answer from https://github.com/kubernetes/community/pull/7678

**Review by castrojo — APPROVED** [review]

**Pull request merged**